public/time.html

```diff
@@ -77,53 +52,32 @@ <h4 style="margin-top: 0px; margin-bottom: 2px;">
       </slide>
 
       <slide ng-if="vis.params.enable_absolut" active="activeSlide.absolute">
-        <h4 style="margin-top: 0px; margin-bottom: 2px;">
-          <strong><i aria-hidden="true" class="fa fa-clock-o"></i> Absolute</strong>
-        </h4>
         <form ng-submit="setAbsolute()">
           <div style="display: inline-block;">
-            <label class="small" style="float:left;">From: <span ng-show="!time.absolute_from"><i>Invalid Date</i></span>
+            <strong><i aria-hidden="true" class="fa fa-clock-o"></i> Absolute search: </strong>
+          </div>
+          <div style="display: inline-block;">
+            <label class="small" style="float:left;">
+              From: <span ng-show="!time.absolute_from"><i>Invalid Date</i></span>
             </label>
             <input type="text" required class="form-control" input-datetime="{{format}}" ng-model="time.absolute_from">
           </div>
 
           <div style="display: inline-block;">
-            <label class="small" style="float:left;">To: <span ng-show="!time.absolute_to"><i>Invalid Date</i></span>
+            <label class="small" style="float:left;">
+              To: <span ng-show="!time.absolute_to"><i>Invalid Date</i></span>
             </label>
             <input type="text" required class="form-control" input-datetime="{{format}}" ng-model="time.absolute_to">
           </div>
 
-          <div style="display: inline-block;">
-            <button class="btn btn-primary" style="margin-bottom: 60px;" ng-disabled="time.absolute_from > time.absolute_to || !time.absolute_from || !time.absolute_to" type="submit">
+          <div style="display: inline-block;"><br>
+            <button class="btn btn-primary" ng-disabled="time.absolute_from > time.absolute_to || !time.absolute_from || !time.absolute_to" type="submit">
               Go
             </button>
             <span class="small" ng-show="time.absolute_from > time.absolute_to"><strong>From</strong> must occur before <strong>To</strong></span>
           </div>
         </form>
       </slide>
-
-      <slide ng-if="vis.params.enable_animation">
```

Comment:
There is no need to permanently remove the time animation since there is an option to disable it